### ISSUE TITLE
Feature: change executable name from SquareDeskPlayer to SquareDesk

### ISSUE THREAD

**Issue by mpogue2** [mpogue2/SquareDesk#163]
Now that we have Lyrics Editing, I think we're satisfying my original vision of "an app where a caller can write and perform square dance music", sort of like a "writing desk" (yeah, that's where it came from, plus all the other good names were taken... :-).  The name SquareDeskPlayer was useful, I think, when the app was indeed just a music player.  Now that it's not anymore, I think it's time to change the executable name back to the original simpler name: SquareDesk.  

This might need some makefile work on all three platforms.

**Comment by danlyke** [comment]
Holler when, I think it'll be relatively easy on Linux.

**Comment by mpogue2** [comment]
Dan, I'd like to make this change, and maybe change test123 to "squaredesk" at the same time.
What's the best way to rename a directory in git?

**Comment by danlyke** [comment]
"git mv", I think.

**Comment by mpogue2** [comment]
Fixed by a4f0f178dbe360bff90c0f3109e55eccf20a11cd .  I decided not to change the name of "test123" directory at this time.  We can always do that later (I think this change is big enough).  Official Name is now "SquareDesk".

@danlyke Holler!  Packaging scripts may need to change.  I know I still have to update the Mac OS installer/packager script, and the Windows installer script will need to change as well.

I also bumped to alpha6, in preparation for another alpha release with the Latest Stuff!
